force-app/main/default/lwc/primaryAffiliationsModalBody/primaryAffiliationsModalBody.html

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="slds-grid slds-grid--vertical slds-grid_vertical-align-start slds-wrap">
+        <template if:true={modifyRecords}>
+            <div class="slds-col slds-size-1-of-1">{labelReference.modalBodyEditSave}</div>
+            <template if:true={accountRecordTypeComboboxVModel}>
+                <div class="slds-col slds-size-1-of-1">
```

Comment:
Let's try and make these comboboxes a little wider to give space for their dropdowns.